x/upgrade/keeper/keeper.go

```diff
@@ -11,6 +11,7 @@ import (
 	"github.com/tendermint/tendermint/libs/log"
 	tmos "github.com/tendermint/tendermint/libs/os"
 
+	"github.com/cosmos/cosmos-sdk/baseapp"
```

Comment:
Instead of importing baseapp can we duplicate the `ProtocolManager` interface here following the expected keepers pattern?